lib/SPIRV/SPIRVTypes.cpp

```diff
@@ -28,11 +29,77 @@ using namespace mlir::spirv;
 // Pull in all enum utility function definitions
 #include "mlir/SPIRV/SPIRVEnums.cpp.inc"
 
+//===----------------------------------------------------------------------===//
+// CompositeType
+//===----------------------------------------------------------------------===//
+
+struct spirv::detail::CompositeTypeStorage : public TypeStorage {
```

Comment:
I think we can remove this and use DefaultTypeStorage.

Reply:
Removed